### ISSUE TITLE
The ingress controller is unable to handle a large list of IP addresses

### ISSUE THREAD

**Issue by uint** [jcmoraisjr/haproxy-ingress#361]
Hello!

When you provide a long list of IPs to whitelist through ingress annotations, haproxy crashes upon being reloaded.

```
2019/07/29 10:57:54 controller-haproxy.go:150: 14: ERR transactionId=7bb0ca3b-59b5-4acc-8420-af3a8520cb46                                                                                                                                                                       
msg="parsing [/tmp/haproxy/haproxy.cfg.7bb0ca3b-59b5-4acc-8420-af3a8520cb46:16]: line too long, truncating at word 65, position 920: <192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50
.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50
.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 192.168.50.1/24 }>."      
line=16 msg="error detected while parsing an 'http-request allow' condition missing closing '}' in condition."
```

This stems from the fact haproxy itself has a limit to how many words a single line of haproxy.cfg can include.

I've tested around and a valid workout is to put the whitelist in a separate file and use the `{ src -f /path/to/whitelist.lst }` syntax in the config file (assuming `whitelist.lst` is the file with CIDRs separated by newlines).

Would it be desirable to make the Ingress controller do it that way rather than putting the IP ranges inline?

**Comment by jcmoraisjr** [comment]
Hi, sure, this is precisely the plan to v0.8. This won't be fixed in snapshot.3 but will in the first beta. I'll have a look also to port this to v0.7.

**Comment by jcmoraisjr** [comment]
Fixed since the first v0.8 beta. This however cannot be moved to v0.7 due to the differences in their code base.
